.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,89 @@
+name: Bug Report
+description: Report an issue or unexpected behavior that occurs within the application
+title: "[Bug]: "
+labels: "bug"
+body:
+  - type: checkboxes
+    attributes:
+      label: Guidelines
+      description: Please ensure you've completed all of the following.
+      options:
```

Comment:
You should add - "Make sure you are on the latest release of FreeTube before submitting an issue"

Reply:
We didnt put it there on purpose. Check out lines 32-36 and line 87. This is enough information for us to know if they are using the latest freetube release

Reply:
@efb4f5ff-1298-471a-8973-3d47447115dc This proposed line would be more about reminding the user than providing us new info.

Reply:
ah ok where do u want it above the line or under the line where u need to check the issue tracker

Reply:
fixed https://github.com/FreeTubeApp/FreeTube/pull/1593/commits/cd65ff0f15f3819f65f37e6d9056dfb96b53619d